core/src/main/java/org/testcontainers/dockerclient/LogToStringContainerCallback.java

```diff
@@ -0,0 +1,24 @@
+package org.testcontainers.dockerclient;
+
+import com.github.dockerjava.api.model.Frame;
+import com.github.dockerjava.core.command.LogContainerResultCallback;
+
+public class LogToStringContainerCallback extends LogContainerResultCallback {
```

Comment:
Am I right in thinking that this exists because we can't use our container logging abstraction (with `ToStringConsumer`) this early in the startup process? 

If that's the case, should we reduce visibility of this class from public so as to reduce confusion with our higher-level logging API?